auth/src/test/java/org/aerogear/mobile/auth/user/UserPrincipalImplTest.java

```diff
@@ -46,6 +43,11 @@ public void testHasRealmRoleSucceeds(){
         assertEquals(userPrincipalImpl.hasRealmRole("rRole"), true);
     }
 
+    @Test
+    public void testToStringImpl(){
+        assertEquals("UserPrincipalImpl{username='test-user', email='null', roles=[cRole, rRole]}", userPrincipalImpl.toString());
```

Comment:
@rachael-oregan when adding this test to the existing code base... the roles=... part was empty